himin/day5/제일 작은 수 제거하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(arr) {
+    if (arr.length === 1) {
+        return [-1];
+    } else {
+        const minValue = Math.min(...arr); 
+        return arr.filter(num => num !== minValue); 
+    }
+}
```

Comment:
```suggestion
function solution(arr) {
    if (arr.length === 1) {
        return [-1];
    }
    
    const minValue = Math.min(...arr); 
    
    return arr.filter(num => num !== minValue);
}
```
@yihimin  `early return` 문 너무 좋은데요 :) 이런 경우 하단 else문은 사실상 불 필요해집니다.
예를 들어 arr.length 가 1인 경우에 첫 번째 return문에 걸리고 함수는 종료 될 것이고 그 이후 로직은 실행되지 않을 거에요. 그 외의 경우에는 모두 2번째 return문에 걸리기 때문에 사실상 else문이 필요없어집니다 :) 

https://www.howdy-mj.me/javascript/early-return

Reply:
@ssi02014 알기 쉬운 설명 감사합니다! 메모해놓겠습니다!